global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "9.0.100-preview.5.24307.3",
+    "version": "8.0.300",
```

Comment:
I don't think we can downgrade this number. (even there might be bugs in the current sdk)
Roslyn (main) change would flows to sdk (main) so it needs to be on 9.x